nuxeo-coldstorage-web/i18n/messages.json

```diff
@@ -1,4 +1,7 @@
 {
+  "documentContentView.infobar.availableToBeRetrieved": "This is a low resolution preview. The original file can be requested by ",
```

Comment:
I am wondering if the message is complete `The original file can be requested by ...` something missing after `requested by `